workflows/MODEL_TO_DATA.nf

```diff
@@ -17,10 +19,12 @@ assert params.email_with_score in ["yes", "no"], "Invalid value for ``email_with
 params.cpus = "4"
 // Default Memory to dedicate to RUN_DOCKER
 params.memory = "16.GB"
-// Scoring Script
-params.scoring_script = "model_to_data_score.py"
-// Validation Script
-params.validation_script = "validate.py"
+// The container that houses the scoring and validation scripts
+params.challenge_container = "ghcr.io/jaymedina/test_model2data:latest"
```

Comment:
This container should probably be added as a package in this repository along with the Dockerfile and scripts being added to the repository for maintainability purposes.

Reply:
We are using this one temporarily but I don't plan to make it permanent; I don't think we should have the testing container belong to an individual user (maintenance issues, as I think you were implying). I eventually plan on replacing this one with the template container that Verena makes.